src/components/Translatathon/ApplyNow.tsx

```diff
@@ -1,8 +1,8 @@
-import { Box, Flex } from "@chakra-ui/react"
-
 import { ButtonLink } from "@/components/Buttons"
```

Comment:
We need to use the [new ButtonLink component](https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/ui/buttons/Button.tsx#L135) in this case.

Reply:
made required changes